.circleci/real_config.yml

```diff
@@ -70,24 +68,31 @@ release-and-rc-filters: &release-and-rc-filters
       - /.*/
   tags:
     only:
-      - /(\d)+(\.(\d)+)+/
-      - /((\d)+(\.(\d)+)+)(-rc)(\d)+/
+      - /v\d+\.\d+\.\d+(rc\d+)?/
 
 rc-filters: &rc-filters
   branches:
     ignore:
       - /.*/
   tags:
     only:
-      - /((\d)+(\.(\d)+)+)(-rc)(\d)+/
+      - /v\d+\.\d+\.\d+rc\d+/
```

Comment:
this would be a great place for a yaml anchor considering you could give this pattern a name

Reply:
Ibid.